lib/tools/te_enum.h

```diff
@@ -296,6 +296,50 @@ extern int te_enum_translate(const te_enum_trn trn[],
                              int value, te_bool reverse,
                              int unknown_val);
 
+/**
+ * A mapping between two bitmasks corresponding to each other via
+ * named bits.
+ *
+ * This structure permits the mapping of one or more bits from one
+ * side to one or more bits on the other side, with the constraint
+ * that the bits on each side do not overlap. This constraint results
+ * in the prohibition of mapping two or more set bits to the same
+ * corresponding bits on the other side.
+ *
+ * An array of mappings should end with #TE_ENUM_BITMASK_CONV_END and
+ * should not contain #UINT64_MAX as a value of one of the fields.
```

Comment:
`@c UINT64_MAX` because it's a system constant which cannot be cross-references.